spec/services/location_lookup_service_spec.rb

```diff
@@ -1,4 +1,3 @@
-require "test_helper"
 require "gds_api/test_helpers/mapit"
 
 describe LocationLookupService do
```

Comment:
`RSpec.describe` 
The parser should do it but doesn't, and I didn't spend the time to figure out why sorry!